pandas/tests/test_lib.py

```diff
@@ -184,6 +185,52 @@ def test_get_reverse_indexer(self):
         self.assertTrue(np.array_equal(result, expected))
 
 
+class TestNullObj(tm.TestCase):
+
+    def test_basic(self):
+        arr = np.array([1, None, 'foo', -5.1, pd.NaT, np.nan])
+        expected = np.array([False, True, False, False, True, True])
+
+        result = lib.isnullobj(arr)
+        tm.assert_numpy_array_equal(result, expected)
+
+    def test_non_ndarray_inp(self):
+        ind = pd.Index([1, None, 'foo', -5.1, pd.NaT, np.nan])
```

Comment:
But they're unit tests for two different methods, one of which is a helper for another.  Sure they might be similar, but from an organisation perspective, IMO it's easier to put the tests next to the module whose methods / functions you are testing.  That is an easier way of searching AND an easier way of organising.